src/runtime/runtime.ts

```diff
@@ -345,11 +361,7 @@ export class TestnetRuntime extends Runtime {
       ({} as any), // ignored
       this.config.helperUrl!
     );
-    if (this.config.masterAccount) {
-      throw new Error('custom masterAccount not yet supported on testnet ' + this.config.masterAccount);
```

Comment:
Now that we're only checking `if (!masterAccount)`, will it fail silently if someone DOES provide `masterAccount`?

Reply:
It should still work.  Haven't tested it though.  The idea is that the if the account doesn't exist then we go to account creator.  Though I think we should make another class that handles accounts/cleanup.